pywbemtools/pywbemcli/pywbemcli.py

```diff
@@ -125,6 +134,51 @@ def execute_startup_script(file_path, verbose):
             raise click.Abort()
 
 
+def _resolve_mock_server(mock_server):
+    """
+    Validate and resolve the mock_server parameter.
+    Validates that all files defined in the server exist and that the
+    types are valid.
+
+    Parameters:
+
+      mock_server (tuple of string):
+        Strings defining all of the files that compromise the mock_server
+        definiton.
+
+    Returns: list of resolved mock server definition components derived
+       from the input parameter mock_server.
+
+    Raises:
+      click.ClickException for any errors.
+    """
+    resolved_mock_server = []
+    assert isinstance(mock_server, tuple)
+    # Normalize paths for different OS's
+    mock_server_path = [os.path.normpath(fn) for fn in mock_server]
```

Comment:
On your use of `os.path.normpath()`: Its description says "This string manipulation may change the meaning of a path that contains symbolic links.". So while I like the idea, it is potentially problematic.

Why do we do anything at all? The user ought to specify a valid path name. If the file path references a file then it works and if not then it fails. That works on all OS types with all variations on how file paths can be specified. The only downside of non-normalized file paths is that the caching mechanism will not consider two file paths as equal that reference the same file in different ways. But that is only a very small problem, IMO.

Reply:
1. We need to clarify for the user what the path must be.

2. We want to put put the files into the connecitons file with normpath but with forward slashes so the file is portable between systems.

Reply:
Did it slightly different.  We normpath when we get from user and again when we upload from yaml. rather than manually playing with the forward and backward slashes.